sklearn/svm/src/liblinear/linear.cpp

```diff
@@ -70,6 +79,52 @@ static void info(const char *fmt,...)
 static void info(const char *fmt,...) {}
 #endif
 
+// New random number generator replacing `rand()`, to ensure the same behaviour on windows-linux, with increased speed
+// - (1) Init a `mt_rand` object
+#if INT_MAX == 0x7FFFFFFF
+std::mt19937 mt_rand(std::mt19937::default_seed);
+#elif INT_MAX == 0x7FFFFFFFFFFFFFFF
+std::mt19937_64 mt_rand(std::mt19937::default_seed);
+#else
+info("Random number generator is not fixed for this system. Please report issue. INT_MAX=%d\n", INT_MAX);
+exit(1);
+#endif
+
+// - (2) New public `set_seed()` function that should be used instead of `srand()` to set a new seed.
```

Comment:
I usually don't like "new" in comments as the comment isn't new for long. I understand here that new means "in scikit-learn, but not in liblinear". Could we make that a little clearer in the comment?

Reply:
I'll do this yes